tests/lua/type_mismatch.lua

```diff
@@ -121,6 +126,9 @@ local function generate_tests()
 
             --  effil.gc.step
             test.type_mismatch.input_types_mismatch_p(1, "number", "gc.step", type_instance)
+
+            --  effil.cache.capacity
```

Comment:
obvious comments overblows  code.

Reply:
Hmm, I can't agree with you. As for me such comments visualize the blocks of code. Maybe leave it?)